tests/genericAPI.js

```diff
@@ -5,350 +5,353 @@
  * For full license text, see LICENSE.txt file in the repo root or
  * https://opensource.org/licenses/BSD-3-Clause
  */
+/* todo
+ * come up with replacement tests
+ */
 
-const expect = require('chai').expect;
-const rewire = require('rewire');
-const bdkServer = rewire('../refocus-bdk-server.js');
+// const expect = require('chai').expect;
```

Comment:
might be better just remove those if they are not valid

Reply:
@kriscfoster whats your opinion on this ?

Reply:
Yeah I think removing would be a good idea, let's make all tests have some sort of value